builders/manage/dapp/xcm-channels.md

```diff
@@ -17,15 +17,7 @@ In this guide, you'll learn how to use the [Tanssi dApp](https://apps.tanssi.net
 
 ## Accessing the Channel Management Panel {: #accesing-channel-management-panel }
 
-The Tanssi dApp makes it easy for the appchain governor to manage cross-chain communication channels. It allows requesting new channels, accepting incoming channel opening requests, closing existing channels, and more. To access the channel management panel, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, and then: 
-
-1. Click on the **Manage** button
-2. Click on the **XCM** button.
-
-![Accessing the channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-1.webp)
-
-!!! note
-    If you don't see your appchain's details on the dashboard, make sure to comply with the [prerequisites](#checking-prerequisites).
+--8<-- 'text/builders/manage/dapp/cross-chain-config-panel.md'
```

Comment:
Maybe I'm missing it, but where else is this used?

Reply:
It's also used in the manage XCM channels guide